src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -11498,6 +11498,15 @@ void sofia_handle_sip_i_invite(switch_core_session_t *session, nua_t *nua, sofia
 			}
 		}
 
+		if (sip->sip_identity) {
+			sip_identity_t *id;
+			for (id = sip->sip_identity; id; id = id->id_next) {
```

Comment:
We are about to merge this PR. Can you please make a new line after the declaration? This will meet our coding guidelines.

Reply:
@hnimminh GitHub checks are green. So let's do the new line and merge.

Reply:
@andywolk  done!